rancher/rancher
# [v2.9] refactor: Fix references as part of migrating from old SDK to new SDK

## Issue: https://github.com/rancher/aks-operator/issues/132

 
## Problem

The API endpoints that are called by the UI when creating or editing AKS clusters were using a very old Azure SDK.
 
## Solution

This PR bumps the version of the Azure SDK being used by the API endpoints.
 
## Testing


## Engineering Testing
### Manual Testing

- Create a new AKS cluster and ensure the UI dropdowns show the correct data (e.g. locations or virtual networks).
- Edit an existing AKS cluster and ensure the UI dropdowns show the correct data (e.g. available AKS upgrade versions).

### Automated Testing

* Test types added/modified:
    * None
* If "None" - Reason: The changes introduced were mostly about calling the new Azure SDK.
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

## QA Testing Considerations

 
### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_
